test/cypress/e2e/integration/app-frontend/summary.js

```diff
@@ -91,7 +91,7 @@ describe('Summary', () => {
       });
   });
 
-  it('is possible to view summary of repeating group', () => {
+  it.only('is possible to view summary of repeating group', () => {
```

Comment:
@olemartinorg This should be removed 🙂

Reply:
Always such a great eye, @haakemon! I added a new PR to fix this, in #333, and hopefully prevented such issues in the future (it.only is great when isolating a test issue, but I've seen it being committed to git before by mistake - should be easy to catch in a pre-commit hook).